src/main/java/com/github/stickerifier/stickerify/logger/SubstringHighlighter.java

```diff
@@ -22,10 +23,10 @@
 public class SubstringHighlighter extends MessageConverter {
 
 	private static final String START_YELLOW = changeColorTo(BOLD + YELLOW_FG);
-	static final String CONTINUE_PREVIOUS_COLOR = changeColorTo("0;" + DEFAULT_FG);
+	static final String CONTINUE_PREVIOUS_COLOR = changeColorTo(RESET + DEFAULT_FG);
 	static final String HIGHLIGHTED_NEW_USER = " " + START_YELLOW + NEW_USER.substring(1) + CONTINUE_PREVIOUS_COLOR;
 	static final String START_GREEN = changeColorTo(BOLD + GREEN_FG);
-	private static final Pattern MIME_TYPE_PATTERN = Pattern.compile("[\\w-]+/[\\w-]+");
+	private static final Pattern MIME_TYPE_PATTERN = Pattern.compile("( )([\\w-]+/[\\w-]+)( )");
```

Comment:
No need to capture the spaces around the primary capture group
```suggestion
	private static final Pattern MIME_TYPE_PATTERN = Pattern.compile(" ([\\w-]+/[\\w-]+) ");
```

Reply:
At one point it was defined like that but I changed it mid rework, I will fix it soon 💯